lib/jekyll/watcher.rb

```diff
@@ -18,12 +18,13 @@ module Watcher
     # Returns nothing.
     def watch(options, site = nil)
       ENV["LISTEN_GEM_DEBUGGING"] ||= "1" if options['verbose']
+      @source = options['source']
 
       site ||= Jekyll::Site.new(options)
       listener = build_listener(site, options)
       listener.start
 
-      Jekyll.logger.info "Auto-regeneration:", "enabled for '#{options["source"]}'"
+      Jekyll.logger.info "Auto-regeneration:", "enabled for #{@source}".green
```

Comment:
This is a module method so I'm wary of using instance variables. Can we pass `options` to the proper method so we can access `options['source']`?